contents/layouts/footers/footer-links/index.tsx

```diff
@@ -46,23 +48,28 @@ const data = [
 const FooterLinks: FC = () => {
   const groups = data.map((group) => {
     const links = group.links.map((link, index) => (
-      <Link
-        key={index}
-        href={link.link}
-        onClick={(event) => event.preventDefault()}
-        fontSize="sm"
-      >
-        {link.label}
-      </Link>
+      <ListItem key={index}>
+        <Link
+          href={link.link}
+          onClick={(event) => event.preventDefault()}
+          fontSize="sm"
+        >
+          {link.label}
+        </Link>
+      </ListItem>
     ))
 
     return (
-      <VStack gap="xs" key={group.title}>
-        <Text fontSize="lg" fontWeight="semibold">
-          {group.title}
-        </Text>
-        {links}
-      </VStack>
+      <List gap="md" key={group.title}>
+        <ListItem>
+          <Text fontSize="lg" fontWeight="semibold">
+            {group.title}
+          </Text>
+        </ListItem>
+        <ListItem>
+          <List gap="xs">{links}</List>
+        </ListItem>
+      </List>
     )
   })
```

Comment:
Wouldn't the code be easier to read if the processing here were separated into a separate component?